Pratiksha/html/PratikshaA5/A5.css

```diff
@@ -0,0 +1,33 @@
+.bg{
+    background-image: url(https://tse3.mm.bing.net/th?id=OIP.mUc57NHscnXINcXYOp--WQHaEK&pid=Api&P=0&w=292&h=165);
+}
+
+.whole{
+   
+    border: 2px solid black;
+}
+.about{
+    color: rgb(209, 109, 21);
+    border-top: 2px solid black;
+    float: right;
+    font-style: italic;
+    font-family:Impact, Haettenschweiler, 'Arial Narrow Bold', sans-serif;
+    font-size: x-large;
+}
+.orange{
+    width:92%;
+    background-color: rgb(209, 109, 21);
+    box-shadow: 1px 2px black;
+    margin-left: -2px;
+   
```

Comment:
this line also remove

Reply:
I want to overlap that paragraph thats why I had put margin